### PR TITLE
Add more formatting options to .astyle

### PR DESCRIPTION
**Pull request by 64**
What do we think all think of these changes? I added small descriptions of each option as a comment, but you can see the full `astyle` documentation [here](http://astyle.sourceforge.net/astyle.html).

And by the way, I'm not saying that these style options are necessarily the best but I do think it is important to maintain a level of consistency throughout the codebase.

### CODE REVIEW

File: .astyle [hunk 1]
```diff
@@ -1,4 +1,13 @@
--A1    # Allman brace style
--k2    # Pointers in the middle
--t     # Tab indent
--r     # Recursive
+# See http://astyle.sourceforge.net/astyle.html
+-A1                   # Allman brace style
+-k2                   # Pointers in the middle
+-t                    # Tab indent
+-j                    # Add braces to one line if/while blocks
```
Comment (Bfgeshka) [inline .astyle]: Should we really use `-j` option? In Allman style it adds 2 lines just because it can. Probably set `-xj` here.

File: .astyle [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+-p                    # Pad operators and commas with one space
+-H                    # Insert a space between 'if'/'while' keywords and parentheses
+-o                    # Keep one line statements
+--max-code-length=120 # Break after 120 chars
+--mode=c              # Parse as C
```
Comment (Bfgeshka) [inline .astyle]: `--mode=c` is default astyle option and is redundand

File: .astyle (hunk 1 above)
Comment (64) [inline .astyle]: Agreed.

File: .astyle (hunk 2 above)
Comment (64) [inline .astyle]: Ok, will remove.

**Comment by Bfgeshka** [comment]
So far so good.

**Pull request merged**